LAn_Number_1-2/Смирнов1-2(2)/Vector3D.cpp

```diff
@@ -65,7 +53,7 @@ double Vector3D::get_z()const
 }
 bool Vector3D::operator==(const Vector3D& other)
 {
-	if (get_vector_length() == other.get_vector_length())
+	if (fabs(this->get_vector_length() - other.get_vector_length()) < numeric_limits<double>::epsilon())
```

Comment:
`return abs(this->get_vector_length() - other.get_vector_length()) < numeric_limits<double>::epsilon();`